source/functions/BEMIO/plotAddedMass.m

```diff
@@ -20,16 +20,16 @@ function plotAddedMass(dofList, varargin)
         'structures when calling: plotAddedMass(hydro1, hydro2, ...)']);
 end
 
-subtitleStrings = getDofNames(dofList);
+varargin = checkAndFormatPlotVars(varargin);
 
-figHandle = figure('Position',[50,500,975,521]);
+figHandle = figure('Position',[50,500,325*size(varargin{1}.plotDofs,1),520]);
```

Comment:
Related to the annotations, perhaps there's a way to make the annotation text wrap automatically? Or a command to always allow space below the plots for whatever size the annotation is? If not, we could also break up bullet points onto an additional line to decrease the horizontal spacing required to read them.

I recognize this is increasing your scope on this PR, but I think this will make maintenance easier in the future.

Reply:
Okay, I got rid of the position specification here as I think it doesn't help much, but left the subplot position in formatPlot.